README.md

```diff
@@ -404,6 +404,71 @@ sudo hab svc load core/artifactory
+
+#### Database Backups
+
+The process of backing up Builder's PostgreSQL database should be no different than backing up any other PostgreSQL database. The process amounts to a pg_dump. If you have a backup strategy for othes production instances of PostgreSQL we suggest you use the same strategy here with the `builder` database. Otherwise the process of a manual backup would be as follows:
+
+1. Shut down the API to ensure no active transactions are occuring. (Optional but preferred)
+        `hab svc stop habitat/builder-api`
+1. Switch to user `hab`
+        `sudo su - hab`
+1. Find your postgres password 
+        `sudo cat /hab/svc/builder-api/config/config.toml`
+1. Export as envvar 
+        `export PGPASSWORD=<pw>`
+1. Run pgdump 
+        `/hab/pkgs/core/postgresql/<version>/<release>/bin/pg_dump --file=builder.dump --format=custom --host=<ip_of_pg_host> --dbname=builder`
+1. Start the api and verify 
+        `sudo hab svc start habitat/builder-api`
+
+Now the backup of your database has been dumped into the `builder.dump` file on your filesystem and can be moved or stored in whatever manner you see fit. For the backup to be useful in a DR scenario we obviously suggest moving the database dump somewhere else. An AWS S3 bucket would suffice for many but again we suggest you store the dump with the same strategy in which you store any other database dumps.
+
+#### Database Restore
+
+Much like the backup process, there is no magic in the database restore for builder. As such, if you already have a restoration strategy your organization is using, feel free to continue using for the `builder` database. Otherwise, a manual restore should be performed as follows:
+
+1. Switch to user `hab`
+        `sudo su - hab`
+1. Find your postgres password
+        `sudo cat /hab/svc/builder-api/config/config.toml`
+1. Export as envvar
+        `export PGPASSWORD=<pw>`
+1. Create the new builder database *
+        `/hab/pkgs/core/postgresql/<version>/<release>/bin/createdb -w -h <url_of_pg_host> -p <configured_pg_port> -U hab builder`
+1. Verify connectivity to the new database instance
+        `/hab/pkgs/core/postgresql/<version>/<release>/bin/psql --host=<url_of_pg_host> --dbname=builder`
+1. Restore the dump into the new DB
+        `/hab/pkgs/core/postgresql/<version>/<release>/bin/pg_restore --host=<url_of_pg_host> --dbname=builder builder.dump`
+1. Start the on-prem builder services
+
+        * In some cases your version of postgres might not have a createdb binary in which case you'll want to connect to database to run the create db command
+
+Just like that your database data should be restored and ready for new transactions!
+
+#### Artifact Backups
+The process of artifact backups is quite a bit more environmentally subjective than postgres if only because we support more than one artifact storage backend. For the sake of these docs we will focus on minio backups.
```

Comment:
Capitalize Postgres

Reply:
What does "environmentally subjective" mean?

Reply:
It means that there is no single process because the process would be determined by the user environment.